jjb/dynamic/scale-ci_install_azure.yml

```diff
@@ -0,0 +1,311 @@
+        name: AZURE_REGION
+    - string:
+        default: ""
+        description: Base domain for OCP install.
+        name: OPENSHIFT_BASE_DOMAIN
+    - string:
+        default: ""
+        description: openshift cluster name.
+        name: OPENSHIFT_CLUSTER_NAME
+    - string:
+        default: "3"
+        description: openshift master node count.
+        name: OPENSHIFT_MASTER_COUNT
+    - string:
+        default: "5"
+        description: openshift worker node count.
+        name: OPENSHIFT_WORKER_COUNT
+    - string:
+        default: "m5.large"
+        description: openshift master instance type.
+        name: OPENSHIFT_MASTER_INSTANCE_TYPE
+    - string:
+        default: "m5.large"
+        description: openshift worker instance type.
+        name: OPENSHIFT_WORKER_INSTANCE_TYPE
+    - string:
+        default: "64"
+        description: openshift master root volume size.
+        name: OPENSHIFT_MASTER_ROOT_VOLUME_SIZE
+    - string:
+        default: "gp2"
+        description: openshift master root volume type.
+        name: OPENSHIFT_MASTER_ROOT_VOLUME_TYPE
+    - string:
+        default: "0"
+        description: openshift master root volume iops
+        name: OPENSHIFT_MASTER_ROOT_VOLUME_IOPS
+    - string:
+        default: "64"
+        description: openshift worker root volume size
+        name: OPENSHIFT_WORKER_ROOT_VOLUME_SIZE
+    - string:
+        default: "gp2"
+        description: openshift worker root volume type
+        name: OPENSHIFT_WORKER_ROOT_VOLUME_TYPE
+    - string:
+        default: "0"
+        description: openshift worker root volume iops
+        name: OPENSHIFT_WORKER_ROOT_VOLUME_IOPS
+    - string:
+        default: "10.128.0.0/14"
+        description: openshift install cidr
+        name: OPENSHIFT_CIDR
+    - string:
+        default: "10.0.0.0/16"
+        description: openshift install machine cidr
+        name: OPENSHIFT_MACHINE_CIDR
+    - string:
+        default: "172.30.0.0/16"
+        description: openshift install service network
+        name: OPENSHIFT_SERVICE_NETWORK
+    - string:
+        default: "23"
+        description: openshift install host prefix
+        name: OPENSHIFT_HOST_PREFIX
+    - string:
+        default: "600"
+        description: Poll attempts to finish post install before failing
+        name: OPENSHIFT_POST_INSTALL_POLL_ATTEMPTS
+    - bool:
+        default: true
+        description: Enables/disables infra node creation using machinesets
+        name: OPENSHIFT_TOGGLE_INFRA_NODE
+    - bool:
+        default: true
+        description: Enables/Disables dedicated workload node creation using machinesets
+        name: OPENSHIFT_TOGGLE_WORKLOAD_NODE
+    - string:
+        default: "machine.openshift.io"
+        description: Machineset label prefix
+        name: MACHINESET_METADATA_LABEL_PREFIX
+    - string:
+        default: "Standard_D2s_v3"
+        description: Infra node instance type
+        name: OPENSHIFT_INFRA_NODE_INSTANCE_TYPE
```

Comment:
Once https://github.com/openshift-scale/scale-ci-deploy/pull/30 is merged this will change to `OPENSHIFT_INFRA_NODE_VM_SIZE`